### PR TITLE
Fix react error on button in playback controls tooltip

### PR DESCRIPTION
**Pull request by foxymiles**
**User-Facing Changes**
None

**Description**
Fix the react error caused by having a disabled button inside a `Tooltip`.

Screenshot 2023-05-26 at 10 24 32 AM

### CODE REVIEW

**Review by jtbandes — COMMENTED** [review]
Wouldn’t this also have the effect of making it so the button is still clickable? I imagine we want it to be actually disabled?

**Comment by foxymiles** [comment]
> Wouldn’t this also have the effect of making it so the button is still clickable? I imagine we want it to be actually disabled?

There's no click action on this button anyway.

**Review by jtbandes — APPROVED** [review]
The only other change I'd consider making is whether there is a `theme` prop to get the disabled opacity, rather than hard-coding 50%.

File: packages/studio-base/src/components/PlaybackControls/index.tsx [hunk 1]
```diff
@@ -206,7 +216,7 @@ export default function PlaybackControls(props: {
               >
                 <HoverableIconButton
                   className={classes.dataSourceInfoButton}
```
Comment (2metres) [inline packages/studio-base/src/components/PlaybackControls/index.tsx:218]: I'd prefer we use a className here something like 
```
className={cx(classes.dataSourceInfoButton, { [classes.disabled]: presence !== PlayerPresence.PRESENT })}
```

Comment (2metres) [reply packages/studio-base/src/components/PlaybackControls/index.tsx:218]: That way you don't need to call the theme inside

Comment (foxymiles) [reply packages/studio-base/src/components/PlaybackControls/index.tsx]: Yeah I agree that's a cleaner solution.

**Pull request merged**